### PR TITLE
SOC-2352 | As a user, I want to start infinite scroll by clicking a Load More button on the posts page (after the first page of posts)

### PR DESCRIPTION
**Pull request by rafalkalinski**
## Links

https://wikia-inc.atlassian.net/browse/SOC-2352

## Description

As a user, I want to start infinite scroll by clicking a Load More button on the posts page (after the first page of posts)

### CODE REVIEW

File: front/main/app/mixins/discussion-wrapper-component.js [hunk 1]
```diff
@@ -1,35 +1,66 @@
...
+		},
+
+		scrollOn() {
+			Ember.$(window).on(`scroll.${this.id}`, this.didScroll.bind(this));
+		},
+
+		loadNextPage() {
+			this.setProperties({
+				pageNum: this.pageNum + 1,
+				isLoading: true,
```
Comment (bkoval) [inline front/main/app/mixins/discussion-wrapper-component.js]: Alphabet

File: front/main/app/mixins/discussion-wrapper-component.js [hunk 2]
```diff
@@ -44,18 +75,13 @@ export default Ember.Mixin.create(
...
-			Ember.$(window).on(`scroll.${this.id}`, this.didScroll.bind(this));
-		},
-
-		/**
-		 * @returns {void}
-		 */
-		willDestroyElement() {
-			Ember.$(window).off(`scroll.${this.id}`);
+		actions: {
+			loadNextPageAction() {
```
Comment (bkoval) [inline front/main/app/mixins/discussion-wrapper-component.js]: Why not just loadNextPage?

File: front/main/app/mixins/discussion-wrapper-component.js [hunk 3]
```diff
@@ -1,35 +1,66 @@
...
-		pageNum: null,
+		isLoading: false,
+		loadOnScrollEnabled: false,
+		pageNum: 0,
 		postsDisplayed: 0,
+		showLoadMoreButton: true,
 		totalPosts: 0,
 
+		autoScrollingOnObserver: Ember.observer('showLoadMoreButton', function () {
+			if (!this.get('showLoadMoreButton') && this.get('loadOnScrollEnabled')) {
```
Comment (bkoval) [inline front/main/app/mixins/discussion-wrapper-component.js]: Could showLoadMoreButton be computed based on loadOnScrollEnabled? We can either have a button or an infinite scroll, so maybe there's no point of checking both these variables here.

File: front/main/app/styles/module/_discussion.scss [hunk 4]
```diff
@@ -549,6 +543,24 @@ body.discussions {
...
+		color: $discussion-load-more-color;
+		text-align: center;
+		width: 100%;
+	}
+
+	.load-more-button {
+		border-style: solid;
+		border-width: 1px;
+		font-size: $type-min;
+		font-weight: 700;
```
Comment (bkoval) [inline front/main/app/styles/module/_discussion.scss]: $font-weight-bold;

File: front/main/app/mixins/discussion-wrapper-component.js (hunk 3 above)
Comment (rafalkalinski) [inline front/main/app/mixins/discussion-wrapper-component.js]: There was an idea to have both like on Fandom page :)

**Comment by bkoval** [comment]
LGTM 🌴

**Pull request merged**